Client/Frontend/Browser/TabManager.swift

```diff
@@ -39,7 +39,21 @@ class TabManager {
         return tabs[index]
     }
 
+    func getTab(webView: WKWebView) -> Browser? {
+        
+        for i in 0..<count {
+            if tabs[i].webView === webView {
+                return tabs[i]
+            }
+        }
+        return nil
+    }
+
     func selectTab(tab: Browser?) {
+        
+        if tab == nil {
+            return
+        }
```

Comment:
Please remove this change. It's OK for no tab to be selected, and we still want to go through the code below that that fires the delegate when this happens, so we don't want to return early here.